docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.11
+
+RUN apt-get update && apt-get install -y \
+	ffmpeg
+
+RUN git clone https://github.com/s0md3v/roop.git /app
```

Comment:
`/root` instead of `/app`

Reply:
fixed (I hope you mean `/roop` as you do in other comments)